addons/web/static/tests/control_panel/custom_filter_item_tests.js

```diff
@@ -70,6 +70,35 @@ odoo.define('web.filter_menu_generator_tests', function (require) {
             cfi.destroy();
         });
 
+        QUnit.test('custom OR filter presets new condition from preceding', async function (assert) {
+            assert.expect(1);
+
+            const searchModel = new ActionModel();
+            const cfi = await createComponent(CustomFilterItem, {
+                props: {
+                    fields: this.fields,
+                },
+                env: { searchModel },
+            });
+
+            // Add first filter condition
+            await cpHelpers.toggleAddCustomFilter(cfi);
+            await testUtils.fields.editSelect(cfi.el.querySelector('.o_generator_menu_field'), 'color');
+            await testUtils.fields.editSelect(cfi.el.querySelector('.o_generator_menu_operator'), '!=');
+
+            // Add a second conditon on the filter being created
+            await cpHelpers.addCondition(cfi);
+
+            // Check the defaults for field and operator dropdowns
+            const firstColorSelectionValue = this.fields.color.selection[0][0];
+            assert.deepEqual(
+                [...cfi.el.querySelectorAll('.o_filter_condition:nth-of-type(2) .o_input')].map(el => el.value),
+                ['color', '!=', firstColorSelectionValue]
+            );
```

Comment:
I think it would be more readable to have two asserts checking that both color and != are selected options in the second condition (and optionally a third one asserting that the value is unset).